packages/facets-core/src/facet-bars-value/FacetBarsValue.ts

```diff
@@ -257,7 +257,8 @@ export class FacetBarsValue extends FacetBlueprint {
                         styles.push(kBarStyleGenerators[kBarStyleSuffixes[ii]](hostTheme, i, optionValue));
                     }
                 }
-            } while (++i < n || hasOption);
+                console.log(`n = ${n}, i = ${i}, hasOption = ${hasOption}`);
```

Comment:
if keeping, should be console.debug

Reply:
(or remove along with the one below)

Reply:
Good catch. That was just debugging. It's been removed.